torch_geometric/nn/conv/message_passing.py

```diff
@@ -85,22 +91,35 @@ class MessagePassing(torch.nn.Module):
         'size_i', 'size_j', 'ptr', 'index', 'dim_size'
     }
 
-    def __init__(self, aggr: Optional[Union[str, List[str]]] = "add",
-                 flow: str = "source_to_target", node_dim: int = -2,
-                 decomposed_layers: int = 1):
+    def __init__(
+        self,
+        aggr: Optional[Union[str, List[str], Aggregation]] = "add",
```

Comment:
We would need some additional `**aggr_kwargs` passed to the `aggregation_resolver`.